Speech/src/V1/Gapic/SpeechGapicClient.php

```diff
@@ -55,7 +55,7 @@
  * ```
  * $speechClient = new SpeechClient();
  * try {
- *     $encoding = RecognitionConfig\AudioEncoding::FLAC;
+ *     $encoding = AudioEncoding::FLAC;
```

Comment:
![](http://www.quickmeme.com/img/83/83206ae67da70662c246b9cca0b92d2da77cae199a14e8d23e102739d5192b2e.jpg)

/cc @hzyi-google

Looks like there could be an issue related to https://github.com/googleapis/gapic-generator/pull/2757

The `Google\Cloud\Speech\V1\RecognitionConfig\AudioEncoding` import is missing.

Reply:
This small sample here does not have an import section at all, the imports above are for the client I think. The type is imported in unit tests and smoke tests.

Reply:
That's a really good point. Based on your experience with the generator, do you think it would be difficult to add imports into the samples as well?

Reply:
I don't think so, considering the imports for standalone samples are already there.